hcfb-fe/src/app/rescue/rescue-add.component.ts

```diff
@@ -0,0 +1,56 @@
+import { Component, OnInit } from '@angular/core';
+import { Rescue } from './Rescue';
+import { Subscription } from 'rxjs/Subscription';
+import { RescueService } from './rescue.service';
+import { Router } from '@angular/router';
+
+@Component({
+    selector: 'rescue-add',
+    templateUrl: './rescue-add.component.ts'
+})
+
+export class RescueAddComponenet implements OnInit {
+    rescueSub: Subscription;
+    test : Date;
+    ngOnInit(){
+        
+    }
+    data : Rescue = { 
+
+    }
+    newRescue: Rescue = {
+        store_name: "testHEB",
+        store_number: "121234321",
+        district: "Buda",
+        location: "2442 developer lane",
+        meat_temp_pick_up: "",
+        meat_temp_drop_off: "",
+        produce_temp_pick_up:"",
+        produce_temp_drop_off:"",
+        created_at: this.test,
+        updated_at: this.test,
+        last_updated_by: " "
+    }
+
+    constructor(private rescueService: RescueService, private router: Router) {};
+
+    getLocation(): void {
```

Comment:
"getLocation()" sounds like a method that's intended to return something, while this initializes the subscription instead. How about something like `subscribeToLocation` instead?